packages/rum-react/README.md

```diff
@@ -0,0 +1,8 @@
+# `rum-react`
+
+> TODO: description
```

Comment:
Are we waiting on the API finalisation before we start working on the docs?

Reply:
Yeah, I will create a separate PR for that.